terraform/environment/api-keys.tf

```diff
@@ -71,3 +71,27 @@ module "trusted_service_access_sirius_production" {
     "arn:aws:iam::649098267436:role/api-ecs-production-2021080908515105620000000a"
   ]
 }
+
+module "trusted_service_access_shared_development" {
+  trusted_service_name               = "shared-development"
+  source                             = "../modules/trusted_service_access"
+  aws_api_gateway_rest_api           = aws_api_gateway_rest_api.kinesis_stream_api_gateway.id
+  aws_api_gateway_stage              = aws_api_gateway_stage.kinesis_stream_api_gateway.stage_name
+  secret_recovery_window_in_days     = 0
+  secretsmanager_api_keys_kms_key_id = aws_kms_key.api_key.key_id
+  identifiers_arns = [
+    "arn:aws:iam::679638075911:root"
+  ]
+}
+
+module "trusted_service_access_shared_production" {
+  trusted_service_name               = "shared-production"
```

Comment:
these keys should be per service. What service is shared is going to use opg-metrics?

Reply:
the keys should be for jenkins rather than for the account then